Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Dockerfile
```

Comment:
Thanks for putting this PR up @katenels!  I created http://dockerhub.com/openedx/blockstore so I believe it should be good to merge if the Dockerhub credentials are available.

A good next step would be to revamp this docker file so that it looks more like what we run in production.  https://github.com/edx/enterprise-catalog/blob/master/Dockerfile may be a good reference. cc @kdmccormick